### ISSUE TITLE
False positive SlevomatCodingStandard.Namespaces.UnusedUses + searchAnnotations + inline `@link` / `@see`

### ISSUE THREAD

**Issue by LastDragon-ru** [slevomat/coding-standard#1303]
```php
<?php declare(strict_types = 1);

namespace Bug;

use Iterator;
use SplFileInfo;
use SplObjectStorage;
use stdClass;

/**
 * Link to the {@link stdClass}, also {@see SplObjectStorage}.
 *
 * @see  Iterator
 * @link SplFileInfo
 */
class Bug {
    // empty
}
```

```xml
<?xml version="1.0"?>
<ruleset name="LaraAspStandard"
         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
         xsi:noNamespaceSchemaLocation="./vendor/squizlabs/php_codesniffer/phpcs.xsd">
    <description>Lara Asp coding standard, based on PSR12</description>
    <config name="installed_paths" value="../../slevomat/coding-standard"/>
    <file>./packages</file>
    <rule ref="SlevomatCodingStandard.Namespaces.UnusedUses">
        <properties>
            <property name="searchAnnotations" value="true"/>
        </properties>
    </rule>
</ruleset>
```

```
----------------------------------------------------------------------
FOUND 2 ERRORS AFFECTING 2 LINES
----------------------------------------------------------------------
 7 | ERROR | [x] Type SplObjectStorage is not used in this file.
 8 | ERROR | [x] Type stdClass is not used in this file.
----------------------------------------------------------------------
PHPCBF CAN FIX THE 2 MARKED SNIFF VIOLATIONS AUTOMATICALLY
----------------------------------------------------------------------
```

_PS: 3c38ab6b2a3ed9f78edfadda11f2fc981d5664fb_

**Comment by kukulich** [comment]
The `@link` does not support this format, see https://docs.phpdoc.org/guide/references/phpdoc/tags/link.html#link

`@see` cannot be supported currently because used parser does not support it: https://github.com/phpstan/phpdoc-parser/tree/master/src/Ast/PhpDoc

**Comment by LastDragon-ru** [comment]
>The @link does not support this format, see https://docs.phpdoc.org/guide/references/phpdoc/tags/link.html#link

Yes, but as you can see in the example `@link SplFileInfo` already works, it is also supported by PhpStrorm.

**Comment by kukulich** [comment]
@LastDragon-ru I'm sorry I read it wrong.

We don't and will not support `{@link stdClass}` or `{@see SplObjectStorage}`.
